life/modernized/10-chapter-10.adoc

```diff
@@ -227,6 +254,10 @@ and the practice of Friends relating to it and other
 things were very weightily and closely discoursed.
 The resolutions were these following:
 
+[.embedded-content-document]
+--
+
+[.numbered]
```

Comment:
This numbered group needs the new syntax... with the [.numbered-group] tag, and the ==== before and after. The other two numbered groups in this chapter are perfect. 👍🏻